### ISSUE TITLE
Can't get widget display to use the icons

### ISSUE THREAD

**Issue by pbosavage** [evoluteur/structured-filter#21]
I'm very interested in using this package,but I am having difficulty getting the widget display right. 
I'm using jQuery 2.2.4, jQuery-ui 1.12.1. 

I'm following the installation instructions per the documentation. I am attempting to use this in a Flask application that uses Jinja templating and bootstrap 3.Rather than getting the nice looking display in the demo, I am getting text rather than the icons, an unappealing looking filter list, and the filter list has no way to remove a filter. In short, the icons and attractive theming is not rendering. I've even tried explicitly setting the buttonLabels option to false. 

<link href="/static/jquery-ui-1.12.1/jquery-ui.css" rel="stylesheet" type="text/css"/>
<link href="/static/jquery-ui-themes-1.12.1/themes/ui-lightness/jquery-ui.css" rel="stylesheet" type="text/css"/>
<link href="/static/jquery-ui-themes-1.12.1/themes/ui-lightness/theme.css" rel="stylesheet" type="text/css"/>
<link href="/static/structured-filter-master/css/structured-filter.css" rel="stylesheet" type="text/css"/>
...
<script src="/static/js/jquery-2.2.4.js"></script>
<script src="/static/jquery-ui-1.2.1/jquery-ui.js"></script>
<script src="/static/bootstrap-3.3.7/dist/js/bootstrap.js" crossorigin="anonymous"></script>
<script src="/static/structured-filter-master/js/structured-filter.js"></script>

Can you assist? Thank you

**Comment by evoluteur** [comment]
There may be incompatibilities between the CSS of jQuery-UI and Bootstrap.
Does it work if you remove the Bootstrap CSS your app?

**Comment by pbosavage** [comment]
Thank you for the quick response. I tried that prior to posting my question - it doesn't change the way the widget is rendered, and bootstrap is a requirement anyway. The UI for the application makes significant use of bootstrap. I also tried using an older version of both jquery-ui and of structured-filter (per the documentation) and the result is the same. Wish I knew what was going wrong - this package looks like exactly what I was looking for!
